pihole-google.txt

```diff
@@ -830,26 +829,461 @@ www.google.tt
 www.google.vg
 www.google.vu
 www.google.ws
+_tcp.google.com
+alt2.aspmx.l.google.com
+alt1.aspmx.l.google.com
+alt4.aspmx.l.google.com
+aspmx.l.google.com
+alt3.aspmx.l.google.com
+_spf.google.com
+_netblocks.google.com
+_netblocks3.google.com
+_netblocks2.google.com
+_tls.google.com
+_caldav._tcp.google.com
+_domainkey.google.com
+_jabber-client._tcp.google.com
+_jabber._tcp.google.com
+alt1.xmpp.l.google.com
+alt3.xmpp.l.google.com
+alt2.xmpp.l.google.com
+alt1.xmpp-server.l.google.com
+alt2.xmpp-server.l.google.com
+alt3.xmpp-server.l.google.com
+xmpp-server.l.google.com
+xmpp.l.google.com
+alt4.xmpp-server.l.google.com
+alt4.xmpp.l.google.com
+_ldap._tcp.google.com
+_xmpp-client._tcp.google.com
+_xmpp-server._tcp.google.com
+blog.google.com
+home.google.com
+groups.google.com
+www.blogger.com
+blogger.l.google.com
+groups.l.google.com
+alt3.gmr-smtp-in.l.google.com
+alt4.gmr-smtp-in.l.google.com
+gmr-smtp-in.l.google.com
+alt2.gmr-smtp-in.l.google.com
+alt1.gmr-smtp-in.l.google.com
+sites.google.com
+www3.l.google.com
+www-wide.l.google.com
+search.google.com
+feeds.google.com
+uploads.google.com
+feedsftp.l.google.com
+www5f.google.com
+admin.google.com
+maven.google.com
+googlecn-lopri.l.google.com
```

Comment:
We discovered that `*.l.google.com` has a lot of subdomains, including random strings. It is impossible to put all these domains in the blocklist. That's why we created a few regexes in addition to the list, you can find them here:

https://github.com/nickspaargaren/no-google#important-regex-filters-not-included-in-blocklist

Please remove all `*.l.google.com` domains from the pull request and optionally add the regex to your blocking software solution.

Reply:
deleted everything that included .l.google.com